pkg/data/mariadb.go

```diff
@@ -0,0 +1,419 @@
+	return nil
+}
+
+func VerifyMariaDBDataCMD(f cmdutil.Factory) *cobra.Command {
+	var (
+		dbName string
+		rows   int
+	)
+
+	mdVerifyCmd := &cobra.Command{
+		Use: "mariadb",
+		Aliases: []string{
+			"md",
+		},
+		Short:   "Verify rows in a MariaDB database",
+		Long:    `Use this cmd to verify data in a mariadb object`,
+		Example: `kubectl dba verify mariadb -n demo sample-mariadb --rows 1000`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				log.Fatal("Enter mariadb object's name as an argument.")
+			}
+			dbName = args[0]
+
+			namespace, _, err := f.ToRawKubeConfigLoader().Namespace()
+			if err != nil {
+				klog.Error(err, "failed to get current namespace")
+			}
+
+			opts, err := newMariaDBOpts(f, dbName, namespace)
+			if err != nil {
+				log.Fatalln(err)
+			}
+
+			tunnel, err := lib.TunnelToDBService(opts.config, dbName, namespace, api.MySQLDatabasePort)
+			if err != nil {
+				log.Fatal("couldn't create tunnel, error: ", err)
+			}
+			defer tunnel.Close()
+
+			err = opts.verifyDataExecCmd(tunnel, rows)
+			if err != nil {
+				log.Fatal(err)
+			}
+		},
+	}
+
+	mdVerifyCmd.Flags().IntVarP(&rows, "rows", "r", 10, "rows in ")
+
+	return mdVerifyCmd
+}
+
+func (opts *mariadbOpts) verifyDataExecCmd(tunnel *portforward.Tunnel, rows int) error {
+	if rows <= 0 {
+		return fmt.Errorf("rows need to be greater than 0")
+	}
+
+	command := ` 
+		USE mysql;
+		SELECT COUNT(*) FROM kubedb_table; 
+	`
+	out, err := opts.executeCommand(tunnel.Local, command)
+	if err != nil {
+		return err
+	}
+
+	output := strings.Split(out, "\n")
+
+	totalKeys, err := strconv.Atoi(strings.TrimPrefix(output[1], " "))
+	if err != nil {
+		return err
+	}
+	if totalKeys >= rows {
+		fmt.Printf("\nSuccess! MariaDB database %s/%s contains: %d keys\n", opts.db.Namespace, opts.db.Name, totalKeys)
+	} else {
+		fmt.Printf("\nError! Expected keys: %d . MariaDB database %s/%s contains: %d keys\n", rows, opts.db.Namespace, opts.db.Name, totalKeys)
+	}
+	return nil
+}
+
+func DropMariaDBDataCMD(f cmdutil.Factory) *cobra.Command {
+	var dbName string
+
+	mdDropCmd := &cobra.Command{
+		Use: "mariadb",
+		Aliases: []string{
+			"md",
+		},
+		Short:   "Verify rows in a MariaDB database",
+		Long:    `Use this cmd to verify data in a mariadb object`,
+		Example: `kubectl dba drop mariadb -n demo sample-mysql`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
+				log.Fatal("Enter mariadb object's name as an argument.")
+			}
+			dbName = args[0]
+
+			namespace, _, err := f.ToRawKubeConfigLoader().Namespace()
+			if err != nil {
+				klog.Error(err, "failed to get current namespace")
+			}
+
+			opts, err := newMariaDBOpts(f, dbName, namespace)
+			if err != nil {
+				log.Fatalln(err)
+			}
+
+			tunnel, err := lib.TunnelToDBService(opts.config, dbName, namespace, api.MySQLDatabasePort)
+			if err != nil {
```

Comment:
MariaDBDatabase port

Reply:
MySQLDatabase port and MariaDBBatabase port are the same. And in apimechinary there are no MariaDBDatabase port. So mehedi vai suggests using this one.